blog/2022-09-15-tauri-1.1.mdx

```diff
@@ -0,0 +1,179 @@
+
+To update the Cargo dependencies, run the following in the `src-tauri` folder:
+
+```
+$ cargo update
+```
+
+## What's in 1.1.0
+
+### Security patch
+
+This release includes a patch for a security vulnerability reported by [@martin-ocasek](https://github.com/martin-ocasek). The `readDir` function was able to return entries outside the configured scope when a symlink is found. The patch is also available in 1.0.6. See [the issue](https://github.com/tauri-apps/tauri/issues/4882) for more details.
+
+### Icon generation
+
+We have been recommending using the [tauricon](https://github.com/tauri-apps/tauricon) project to generate icons for your Tauri application using a single source PNG. Several issues have been reported, and we decided to `Rewrite It In Rust` to enhance its stability. This allowed us to move this functionality to the main Tauri CLI, so now you can use the `tauri icon` command.
+
+### cargo-binstall support for the Tauri CLI
+
+The Tauri CLI can now be installed using [cargo-binstall](https://github.com/cargo-bins/cargo-binstall), a mechanism to download and install prebuilt Rust binaries. The binaries are available for the main targets and can be installed with:
+
+```
+$ cargo install cargo-binstall
+# in the next release, `$ cargo binstall tauri-cli` will be enough
+$ cargo binstall tauri-cli --pkg-url "{ repo }/releases/download/cli.rs-v{ version }/cargo-tauri-{ target }.{ archive-format }"
+$ cargo tauri dev # run any tauri command!
+```
+
+### Create system trays at runtime
+
+The system tray APIs previously only available in `tauri::Builder::system_tray` function can now be used at runtime, giving you control over its lifetime and even create multiple trays.
+
+Here's a quick example on how to use it:
+
+```rust
+use tauri::{Builder, CustomMenuItem, SystemTray, SystemTrayEvent, SystemTrayMenu};
+Builder::default()
+    .setup(|app| {
+        let handle = app.handle();
+        SystemTray::new()
+            .with_id("main")
+            .with_menu(
+                SystemTrayMenu::new().add_item(CustomMenuItem::new("quit", "Quit"))
+            )
+            .on_event(move |event| {
+                let tray_handle = handle.tray_handle_by_id("main").unwrap();
+                if let SystemTrayEvent::MenuItemClick { id, .. } = event {
+                    if id == "quit" {
+                        tray_handle.destroy().unwrap();
+                    }
+                }
+            })
+    });
+```
+
+### TOML configuration support
+
+In the 1.0 releases Tauri supports the `JSON` configuration format by default, and `JSON5` when the `config-json5` Cargo feature is enabled, meaning the following configurations are valid:
+
+```json
+tauri.conf.json:
+{
+  "build": {
+      "devPath": "http://localhost:8000",
+      "distDir": "../dist"
+  }
+}
+```
+
+```json5
+tauri.conf.json5:
+{
+  build: {
+      // devServer URL (comments are allowed!)
+      devPath": http://localhost:8000,
+      distDir": ../dist
+  }
+}
+```
```

Comment:
Need to add json5 in our code formatting, will add it in another commit

~~~suggestion
```json5 title=tauri.conf.json5
{
  build: {
      // devServer URL (comments are allowed!)
      devPath": http://localhost:8000,
      distDir": ../dist
  }
}
```
~~~